EventEnabledInsurance/prereqs.sh

```diff
@@ -234,6 +235,16 @@ $TKN clustertask ls | grep git-clone
 
 echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
 
+echo "INFO: Installing Elastic search operator and Elastic search instance..."
+if ! ${CURRENT_DIR}/setup-elastic-search.sh -n ${namespace} -e $ELASTIC_NAMESPACE; then
+  echo -e "\n$cross ERROR: Failed to install and setup elastic search in the '$namespace' namespace"
```

Comment:
This message (and in the else) is a bit confusing. I guess it installed in the $ELASTIC_NAMESPACE and did some setup in the $namespace? It didn't install in the $namespace.

Reply:
Done